### PR TITLE
Bump golang to 1.23.2 for hello-app-redis

### PR DESCRIPTION
**Pull request by NimJay**
## Description

Please see Google-internal bug: [b/370382662](http://b/370382662)

* [x] Merge this pull-request for me once it is approved.

### CODE REVIEW

File: quickstarts/hello-app-redis/Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.21.0 as builder
+FROM golang:1.21.8 as builder
```
Comment (NimJay) [inline quickstarts/hello-app-redis/Dockerfile:15]:
```suggestion
FROM golang:1.23.2 as builder
```

File: quickstarts/hello-app-redis/Dockerfile [hunk 2]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.21.0 as builder
+FROM golang:1.23.2 as builder
```
Comment (NimJay) [inline quickstarts/hello-app-redis/Dockerfile:15]: Judging from https://go.dev/dl/, 1.23.2 is the most recent 1.23 patch.

**Review by iennae — APPROVED** [review]
LGTM; curious if we need to specify to the minor here

**Comment by NimJay** [comment]
Thanks so much for reviewing this, Jennifer!

Closing. Looks like Renovate Bot took care of this for us in https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/pull/1481 — after we enabled Renovate on Dockerfiles via https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/pull/1480.

**Pull request closed without merging**